test/libsyncengine/update_detection/update_detector/testupdatetreeworker.cpp

```diff
@@ -361,11 +361,11 @@ void TestUpdateTreeWorker::testHandleCreateOperationsWithSamePath() {
         CPPUNIT_ASSERT_EQUAL(ExitCode::Ok, _updateTreeWorker->handleCreateOperationsWithSamePath());
 
 
-        // Duplicate paths but distinct name encodings: success
+        // Duplicate paths but distinct name encodings are not supported and should never reach the snapshot.
```

Comment:
`UpdateTree` instead of `snapshot`?

Reply:
I mean `Duplicate paths with distinct name encodings are not supported and should never reach the snapshot, and therefore should not appear in the updateTree either.`

Done here: [19b51ff](https://github.com/Infomaniak/desktop-kDrive/pull/417/commits/19b51ff52f2ca6122694319bbc9dc14614a7ff38)